application/common/models/Password.php

```diff
@@ -194,17 +197,13 @@ public function getUser()
     }
 
     /**
-     * @param bool $runValidation
-     * @param null $attributeNames
-     * @return bool
+     * @param $attribute
      * @throws ConflictHttpException
      */
-    public function save($runValidation = true, $attributeNames = null)
+    public function checkRecentlyUsed($attribute)
     {
         if ($this->hasAlreadyBeenUsedTooRecently()) {
-            throw new ConflictHttpException('May not be reused yet.', 1542395933);
+            throw new ConflictHttpException('May not be reused yet', 1542395933);
```

Comment:
```suggestion
            throw new ConflictHttpException('You have already used this password, please use a new password', 1542395933);
```

Reply:
This message is swallowed by `idp-pw-api`, and passes on the 409 to the UI with its own translated message.  In fact, as it stands today, the UI overrides the API message with its own.